vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthCombinedServerSelectionFragment.kt

```diff
@@ -91,6 +93,11 @@ class FtueAuthCombinedServerSelectionFragment :
             val userUrlInput = state.selectedHomeserver.userFacingUrl?.toReducedUrlKeepingSchemaIfInsecure() ?: viewModel.getDefaultHomeserverUrl()
             views.chooseServerInput.editText().setText(userUrlInput)
         }
+
+        views.chooseServerInput.editText().selectAll()
+        views.chooseServerInput.editText().requestFocus()
+        val inputMethodManager = views.chooseServerInput.editText().context.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
+        inputMethodManager.showSoftInput(views.chooseServerInput.editText(), InputMethodManager.SHOW_IMPLICIT)
```

Comment:
You could use this extension: https://github.com/vector-im/element-android/blob/main/vector/src/main/java/im/vector/app/core/extensions/View.kt#L28

Reply:
Oh perfect thank you for that hint! Wasn't aware of it being available already but I will update the PR as soon as I can get back at my AS (tomorrow probably).